system/admin/views/config-security.html.php

```diff
@@ -20,6 +20,7 @@
   <hr>
   <p><?php echo i18n('Get_one_here');?>  <a target="_blank" href="https://www.google.com/recaptcha/admin">https://www.google.com/recaptcha/admin</a>
   <p><?php echo i18n('Cloudflare_info');?>  <a target="_blank" href="https://developers.cloudflare.com/turnstile/">https://developers.cloudflare.com/turnstile/</a>
+  <p><?php echo i18n('Friendly_Captcha_info');?>  <a target="_blank" href="https://friendlycaptcha.com/#developers">https://friendlycaptcha.com/#developers</a>
```

Comment:
Consider replacing the URL with this sign up form (https://friendlycaptcha.com/signup/account/free/) since all the user needs is to create an account, Cloudflare's option includes the dev site because after creating an account there are a few extra steps to deploy Turnstile.

Reply:
Understood. You’re correct; this company exclusively offers CAPTCHA services, unlike Cloudflare and Google, which provide a broader range of solutions.

Reply:
I have replaced the URL.